scripts/oauth/token.go

```diff
@@ -132,8 +132,8 @@ func (a *OAuthApp) ServeHTTP(writer http.ResponseWriter, request *http.Request)
 	case request.URL.Path == "/" && request.Method == "GET":
 		// Redirect to the OAuth provider.
 		encState := base64.URLEncoding.EncodeToString([]byte(a.State))
-		url := a.Config.AuthCodeURL(encState, a.Options...)
-		writer.Header().Set("Location", url)
+		u := a.Config.AuthCodeURL(encState, a.Options...)
```

Comment:
can we use url instead unless there is a need for it? I sometimes like shortcuts, but I think it would be easier to read for others if it is spelled out. I think the linter would also complain about short parameter names.

Reply:
I changed the name because the var name `url` collides with the import. I was trying to reduce the chance of a future bug. But it's not a big deal and I can change it back if it's confusing.